### DISCUSSION TITLE
Does this work with Jellyfin MPV Shim?

### DISCUSSION THREAD

**Discussion by Batorian** [bloc97/Anime4K#255] [General]
Basically a casting service for Jellyfin which separates its own MPV client.

**Comment by Tama47** [comment]
Yes, according to this https://github.com/jellyfin/jellyfin-mpv-shim#shader-packs

> Shader packs are a recent feature addition that allows you to easily use advanced video shaders and video quality settings. These usually require a lot of configuration to use, but MPV Shim's default shader pack comes with [FSRCNNX](https://github.com/igv/FSRCNN-TensorFlow) and [Anime4K](https://github.com/bloc97/Anime4K) preconfigured. Try experimenting with video profiles! It may greatly improve your experience.

**Reply by Batorian** [reply]
🙏
